### ISSUE TITLE
Markers not drawn after changing scene file

### ISSUE THREAD

**Issue by hak-tpl** [tangrams/tangram-es#787]
I have changed scene file at runtime by calling  `mapController.loadSceneFile(filePath);`. The scene file is changed successfully while the markers drawn on map are gone. Changing scene file back to previous also doesn't draw the markers again. 

Kindly please comment what can be the posssible issue?

Code used to draw location marker is
```
public void addLocationMarker(LngLat place) {

        Map<String, String> marker = new HashMap<>();
        marker.put("type", "currentlocation");
        locationMarkerLayer.addPoint(place, marker);
        mapController.requestRender();

 }
```

**Comment by tallytalwar** [comment]
Hey

We pushed a fix to master for this issue. Will keep you informed when we
cut a new patched 0.3.2 release having this fix. Some time next week.

**Comment by tallytalwar** [comment]
Fixed in [v0.3.2](https://github.com/tangrams/tangram-es/releases/tag/0.3.2-android)
